src/components/Incorporation/SummaryDefineCompany.vue

```diff
@@ -120,7 +122,8 @@ export default class SummaryDefineCompany extends Vue {
   // for template
   readonly RouteNames = RouteNames
 
-  // Getters
+  @Prop({ default: true }) readonly showCompanyName!: boolean
```

Comment:
I made Company Name optional so I could reuse this component on the Restoration Review Confirm page.

Reply:
This now uses a computed property + a store getter (since there are already getters in here). No more prop.